PyPDF2/merger.py

```diff
@@ -26,6 +26,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 from sys import version_info
+import warnings
```

Comment:
This module uses `warnings` in a number of places throughout the file, and I'm not totally sure how it was working before, but :shrug:

Reply:
It was working due to the `from PyPDF2.generic import *`. An explicit import is better though :+1: